README.md

```diff
@@ -1 +1,102 @@
+{'valkrets': [], 'bild_url_192': [], 'uppgift': [], 'intressent_id_check': []}, 
+{'valkrets': [], 'bild_url_192': [], 'uppgift': [], 'intressent_id_check': []}, 
+{'valkrets': [], 'bild_url_192': [], 'uppgift': [], 'intressent_id_check': []}, 
+{'valkrets': [], 'bild_url_192': [], 'uppgift': [], 'intressent_id_check': []}, 
+{'valkrets': [], 'bild_url_192': [], 'uppgift': [], 'intressent_id_check': []}, 
+{'valkrets': [], 'bild_url_192': [], 'uppgift': [], 'intressent_id_check': []}, 
+{'valkrets': [], 'bild_url_192': [], 'uppgift': [], 'intressent_id_check': []}, 
+{'valkrets': [], 'bild_url_192': [], 'uppgift': [], 'intressent_id_check': []}, 
+{'valkrets': [], 'bild_url_192': [], 'uppgift': [], 'intressent_id_check': []}, 
+{'valkrets': [], 'bild_url_192': [], 'uppgift': [], 'intressent_id_check': []}, 
+{'valkrets': [], 'bild_url_192': [], 'uppgift': [], 'intressent_id_check': []}, 
+{'valkrets': [], 'bild_url_192': [], 'uppgift': [], 'intressent_id_check': []}, 
+{'valkrets': [], 'bild_url_192': [], 'uppgift': [], 'intressent_id_check': []}, 
+{'valkrets': [], 'bild_url_192': [], 'uppgift': [], 'intressent_id_check': []}, 
+{'valkrets': [], 'bild_url_192': [], 'uppgift': [], 'intressent_id_check': []}, 
+{'valkrets': [], 'bild_url_192': [], 'uppgift': [], 'intressent_id_check': []}, 
+{'valkrets': [], 'bild_url_192': [], 'uppgift': [], 'intressent_id_check': []}, 
+{'valkrets': [], 'bild_url_192': [], 'uppgift': [], 'intressent_id_check': []}, 
+{'valkrets': [], 'bild_url_192': [], 'uppgift': [], 'intressent_id_check': []}, 
+{'valkrets': [], 'bild_url_192': [], 'uppgift': [], 'intressent_id_check': []}, 
+{'valkrets': [], 'bild_url_192': [], 'uppgift': [], 'intressent_id_check': []}, 
+{'valkrets': [], 'bild_url_192': [], 'uppgift': [], 'intressent_id_check': []}, 
+{'valkrets': [], 'bild_url_192': [], 'uppgift': [], 'intressent_id_check': []}, 
+{'valkrets': [], 'bild_url_192': [], 'uppgift': [], 'intressent_id_check': []}, 
+{'valkrets': [], 'bild_url_192': [], 'uppgift': [], 'intressent_id_check': []}, 
+{'valkrets': [], 'bild_url_192': [], 'uppgift': [], 'intressent_id_check': []}, 
+{'valkrets': [], 'bild_url_192': [], 'uppgift': [], 'intressent_id_check': []}, 
+{'valkrets': [], 'bild_url_192': [], 'uppgift': [], 'intressent_id_check': []}, 
+{'valkrets': [], 'bild_url_192': [], 'uppgift': [], 'intressent_id_check': []}, 
+{'valkrets': [], 'bild_url_192': [], 'uppgift': [], 'intressent_id_check': []}, 
+{'valkrets': [], 'bild_url_192': [], 'uppgift': [], 'intressent_id_check': []}, 
+{'valkrets': [], 'bild_url_192': [], 'uppgift': [], 'intressent_id_check': []}, 
+{'valkrets': [], 'bild_url_192': [], 'uppgift': [], 'intressent_id_check': []}, 
+{'valkrets': [], 'bild_url_192': [], 'uppgift': [], 'intressent_id_check': []}, 
+{'valkrets': [], 'bild_url_192': [], 'uppgift': [], 'intressent_id_check': []}, 
+{'valkrets': [], 'bild_url_192': [], 'uppgift': [], 'intressent_id_check': []}, 
```

Comment:
These are all the same, looks like duplicate values. Are you sure this is the data you get from the members api?

Reply:
problem debugged.